src/methods/tfts.jl

```diff
@@ -0,0 +1,91 @@
+export TFTS
+"""   
+    TFTS(fϵ::Real)
```

Comment:
What is `fε` ? It is not explained in the docstrng...

Reply:
I intended to make this a draft PR, not to merge it. I must have been in a haze and clicked the wrong button. This is not ready. Sorry about that!